packages/cli/lib/projectStructure.js

```diff
@@ -5,10 +5,23 @@ const { logger } = require('@hubspot/local-dev-lib/logger');
 const { logErrorInstance } = require('./errorHandlers/standardErrors');
```

Comment:
Not related to your PR, but I had forgotten about this file. I wonder if there's a case for moving this into local-dev-lib at some point?

Reply:
I could be mixing thing up, but wasn't the idea here that we moved this _out_ of cli-lib/local-dev-lib because its specific to how the CLI displays the errors thrown by the libraries?

Reply:
re: `./errorHandlers/standardErrors`, yeah you're totally right. I was actually talking about this entire projectStructure.js file though.

Reply:
Ohhh gotcha. Yeah that's a good question. It does seem like it would make sense in local-dev-lib, though could also see it making sense to wait until its needed in another package (like a theoretical vscode extension)

Reply:
Agreed 👍 maybe just something to keep an eye on for now